BUILD.md

```diff
@@ -105,8 +105,7 @@ tools to manage projects in VS IDE.
 
 1. Generate build environment using [CMake]
 
-       mkdir c:\projects\mmex\build
-       cd c:\projects\mmex\build
+       cd c:\projects\mmex
```

Comment:
You don't want build mmex in-place mixing sources with different binary versions - CMake standard way is to build out-of-source, leaving git repo clean.
Your change brakes this example too - `cmake ..` below will not find required `CMakeLists.txt`